events.h

```diff
@@ -202,6 +202,32 @@ char EVT_start_loop(EVTHandler *handler);
  */
 void EVT_exit_loop(EVTHandler *handler);
 
+char EVT_enable_virt(EVTHandler *ctx, struct timeval initTime);
+
+struct VirtClkState *EVT_clk(EVTHandler *ctx);
+
+/**
+ * Get the system's current absolute GMT time.
+ *
+ * @param tv A pointer to the timeval structure where the time gets stored
+ *
+ */
+int EVT_get_gmt_time(EVTHandler *ctx, struct timeval *tv);
+int EVT_get_gmt_time_virt(struct timeval *tv);
```

Comment:
Ideally, I think we would only expose `EVT_get_gmt_time` and `EVT_get_monotonic_time` and they would both do the correct thing based on whether or not virtual time is enabled. This would allow core processes calling libproc to "just work" in terms of time simulation.